chapters/3-measurement.tex

```diff
@@ -37,125 +39,209 @@
+	and maps research design information to those observations.
+	Research variables are time-invariant and 
+	often, but not always, controlled by the research team.
+	Examples include
+	ID variables, sampling status, treatment status, and treatment uptake.} 
+but not include any \textbf{measurement variables}\sidenote{
+	\textbf{Measurement variables:} Data that 
+	corresponds to direct observations of the real world, 
+	recorded sentiments of the subjects of the research 
+	or any other aspect your project is studying. 
+	Research variables are not controlled by the research team
+	and often vary over time.
+	Examples include characteristics of the research subject, 
+	outcome variables, input variables among many others.}.
+Research variables and measurement variables 
+often come from the same source,
+but should not be stored in the same way.
+For example, if you are shared administrative data that both includes 
+information on eligibility to be included in the study (research variable) 
+and information on outcome on the topic of your study (measurement variable) 
+you should first decide which variables are research variables, 
+remove them during the data cleaning (see Chapter 6) 
+and instead store them in your master dataset. 
+It is common that you will have to update 
+your master datasets throughout your project.
+
+The most important function of the master dataset 
+is to be the authoritative source for the project ID.
+This means that all observations listed 
+should be uniquely and fully identified by the included project ID variable.\sidenote{
+	\url{https://dimewiki.worldbank.org/ID\_Variable\_Properties}}
+You should also list all other identifiers used in your project, 
+such as names, addresses, or other IDs used by partner organizations,
+and the master datasets can then serve as 
+the linkage between those identifiers and the project ID.
+Because of this, master datasets must, 
+with very few exceptions, always be encrypted.
+Even when a partner organization has a unique identifier, 
+you should always create a project ID specific to your project only, 
+as you are otherwise not in control over 
+who can re-identify your de-identified dataset. 
+
+You should include all observations ever encountered 
+in your master datasets,
+even if they are not eligible for your study.
+This is because, if you ever need to perform a record linkage such as a fuzzy match
+on string variables like proper names, 
+you will make fewer errors the more information you have.
+If you ever need to do a fuzzy match,
+you should always do that between the master dataset
+and the dataset without an unambiguous identifier.
+You should not do anything with that dataset until
+you have successfully merged
+the project IDs from the master dataset.
+
+With the master datasets as an authoritative source 
+of the project ID and all research variables,
+it serves as an unambiguous method of mapping
+the observations in your study to your research design.
```

Comment:
This one-sentence paragraph is a bit weird. can we merge it to the previous one?